build.gradle

```diff
@@ -345,6 +345,7 @@ project('spring-xd-reactor') {
 	dependencies {
 		compile "io.projectreactor:reactor-core"
 		compile "org.springframework.integration:spring-integration-core"
+		compile "org.springframework.integration:spring-integration-expression"
```

Comment:
Fool me once, shame on you. Fool me twice, shame on me: I'm not falling in your trap again; The groupId/artifactId is wrong here